### PR TITLE
Stitch JHU schema changes into time-series

### PR DESCRIPTION
**Pull request by tiffanychu90**
* JHU has had various schema changes
* Stitch the earlier data together
* Standardize the schema to the 3/24/20 version
* Need to adapt to become ETL

[Reference original GitHub issue in public-health-prototype](https://github.com/CityOfLosAngeles/public-health-prototype/issues/38)

### CODE REVIEW

**Comment by hunterowens** [comment]
:+1: @tiffanychu90 do you want me to move into an etl?

**Comment by ian-r-rose** [comment]
Yeah, I think that is the plan.

**Comment by tiffanychu90** [comment]
I want to pin our reading the csv files from JHU's GitHub repo to commit "277426f5d8e59dc2ebc60cba3cc51809a3149628"

**Comment by hunterowens** [comment]
:fireworks:

File: dags/public-health/covid19/jhu-to-esri.py [hunk 1]
```diff
@@ -71,6 +71,76 @@
...
+    'US Virgin Islands': 'VI', 
+    'United States Virgin Islands': 'VI',
+    'Grand Princess': 'Grand Princess',
+    'Diamond Princess': 'Diamond Princess', 
+    'From Diamond Princess': 'Diamond Princess', 
+    'Diamond Princess cruise ship': 'Diamond Princess'
+}
+
+# reverse the dict
+abbrev_us_state = dict(map(reversed, us_state_abbrev.items()))
```
Comment (hunterowens) [inline dags/public-health/covid19/jhu-to-esri.py]: fun til for me!

Comment (tiffanychu90) [reply dags/public-health/covid19/jhu-to-esri.py]: Yes! It was on that GitHub page I copied from. Fun tidbit!

Comment (ian-r-rose) [reply dags/public-health/covid19/jhu-to-esri.py]: Very nice. I've done this with a dict comprehension, but this is a bit more succinct.

**Pull request merged**